src/mumble/DBus.cpp

```diff
@@ -97,3 +97,11 @@ bool MumbleDBus::isSelfMuted() {
 bool MumbleDBus::isSelfDeaf() {
 	return g.s.bDeaf;
 }
+
+void MumbleDBus::startTalking() {
+    g.mw->on_PushToTalk_triggered(true, 0);
```

Comment:
```suggestion
    g.mw->on_PushToTalk_triggered(true, QVariant());
```

Reply:
Happy to make this change. What does this change do? Not questioning your suggestion, but interested in learning.

Reply:
No problem! The function's second argument is a `QVariant`, but we don't use it:
https://github.com/mumble-voip/mumble/blob/df7d3c716ca25e884d635e7aaeab8ad23342092a/src/mumble/MainWindow.cpp#L2500-L2508

When you pass 0 to it, it's as if you wrote `QVariant(0)`: a `QVariant` is constructed with the value `0`.

Reply:
Gotcha. I had looked at the source of the function, but I wasn't sure what the practical difference was between passing QVariant() or 0. Seems like it's just a matter of passing the empty case so if the second argument is ever used in the future we don't cause some unwanted behavior. 

Is there a reason for this second argument's existance, or is it just a matter of future proofing the function?

Reply:
You're welcome!

> I had looked at the source of the function, but I wasn't sure what the practical difference was between passing QVariant() or 0. Seems like it's just a matter of passing the empty case so if the second argument is ever used in the future we don't cause some unwanted behavior.

Correct, in fact `QVariant` has `isNull()`: https://doc.qt.io/qt-5/qvariant.html#isNull

> Is there a reason for this second argument's existance, or is it just a matter of future proofing the function?

https://github.com/mumble-voip/mumble/blob/df7d3c716ca25e884d635e7aaeab8ad23342092a/src/mumble/MainWindow.cpp#L1141-L1143

https://github.com/mumble-voip/mumble/blob/df7d3c716ca25e884d635e7aaeab8ad23342092a/src/mumble/PTTButtonWidget.h#L21-L22

`PTTButtonWidget` was added in 67ed381d91b7af804d905e68c61a5709885b2eec.

@hacst Do you perhaps remember why `triggered()` has a second (unused) argument?